app/forms/openid_connect_authorize_form.rb

```diff
@@ -108,6 +105,14 @@ def ial
     Saml::Idp::Constants::AUTHN_CONTEXT_CLASSREF_TO_IAL[ial_values.sort.max]
   end
 
+  def aal_values
+    acr_values.filter { |acr| %r{/aal/}.match? acr }
+  end
```

Comment:
Nitpick: Looks like this method was moved from its original location in this file but I don't notice any other changes. Can we move it back to where it was (so it doesn't needlessly register as a change) ?

Reply:
This got moves so the IAL methods and the AAL methods are grouped together. I felt like that would be better for future engineers coming in and looking at the code than having some IAL, then AAL, then more IAL, then more AAL.

Reply:
Agreed!